app/controllers/catalog_controller.rb

```diff
@@ -365,7 +365,10 @@ def raw
                                                 document.languages.present?
                                               }
 
-    config.add_summary_field 'eadid', label: 'Collection ID', accessor: :eadid
+    config.add_summary_field 'collection_unitid_ssm', label: 'Collection ID', accessor: :collection_unitid,
```

Comment:
@cmkpowell should this label be "collection id" or "call number"?

Reply:
@respinos Call number

Reply:
@gkostin1966 once the label has been updated to "Call Number", merge away.